ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1187,3 +1187,44 @@
                       default-excludes: true
                       fingerprint: false
                       only-if-success: false
+
+      - 'cloud-{name}-{test_name}-for-pull-request':
+          test_name: multicluster
+          node: 'antrea-multicluster-node'
+          description: 'This is the {test_name} test for {name} pull request.'
+          white_list_target_branches: [ ]
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-multicluster
+          status_url: --none--
+          branches:
+              - ${{sha1}}
+          builders:
+              - builder-multicluster-e2e
+          trigger_phrase: ^(?!Thanks for your PR).*/test-(multicluster-e2e-cloud).*
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-multicluster-e2e-cloud to re-trigger.
```

Comment:
Does it mean we will have two triggers for multi-cluster e2e? Will we keep the old one? or you will rename this after all is verified?

Reply:
What's the answer of this comment? We are using `test-multicluster-e2e` to trigger related test, why is this a different trigger?

Reply:
When this PR is created, we need to test the multicluster e2e test, in order to avoid conflict with the current E2E test, I added the suffix "cloud" to this one. But Zhengsheng said maybe we could keep the suffix and keep both E2E test, they could be back up for each other.

Reply:
If guys want to use the existing trigger, we can use `test-multicluster-e2e` in this new job, and modify the trigger in the old job to add a `-lab` suffix instead.

Reply:
Yes, I feel let's stick to the well known triggers for main job. It's hard to remember so many phrases, especially if they are named in different fashion (like append "-cloud" to multicluster-e2e only). Backup job could use an extended name and we can use them temporally when the main ones are not available.

Reply:
Thanks for the comments, removed the suffix.